l10n.yaml

```diff
@@ -3,5 +3,4 @@ template-arb-file: app_en.arb
 arb-dir: lib/l10n/arbs
 output-localization-file: app_translations.dart
 output-dir: lib/l10n/generated
-synthetic-package: false
```

Comment:
Why this delete? 
I didn't want translations to be treated as a different package.